source/includes/v1/records_api.md

```diff
@@ -25,7 +25,7 @@ Parameter            | Description
 `pretty_print`       | If set to true (default is false), pretty prints JSON and JSONP output
 `format`             | Format of the response output. Can be `json` (default), `jsonp`, `csv`, `geojson`, `geojsonp`
 `callback`           | JSONP or GEOJSONP callback
-`sort`               | Sorts results by the specified field (in `modified`, `issued`, `created` and `records_count`). By default, the sort is descending. A minus sign `-` may be used to perform an ascending sort
+`sort`               | Sorts results by the specified field (e.g. `record_timestamp`). By default, the sort is descending. A minus sign `-` may be used to perform an ascending sort. Sorting is only available on numeric fields (int, double, date and datetime) and on text fields which have the sortable annotation
```

Comment:
int and double types don't exist in our platform. It must be in my opinion, `numeric fields (Integer, Decimal), date fields (Date, DateTime) and one text fields ...`
https://help.opendatasoft.com/platform/en/publishing_data/05_processing_data/defining_a_dataset_schema.html#choosingtype

Reply:
I agree.